src/components/ResourceList/components/FilterControl/components/DateSelector/DateSelector.tsx

```diff
@@ -341,7 +341,8 @@ class DateSelector extends React.PureComponent<CombinedProps, State> {
       return;
     }
 
-    const nextDate = new Date(userInputDate.replace(/-/g, '/'));
+    const formattedDate = formatDateValue(new Date(userInputDate));
```

Comment:
Nitpick but I feel that `formattedDate` does not reflect completely the intention here. Same for `formatDateValue`.

Maybe the function should be called `formatDateForTimezone`?
And your const `formattedDateForTimezone`?

Reply:
That makes sense, I'll fix it